### DISCUSSION TITLE
Styling Text Section

### DISCUSSION THREAD

**Discussion by popados** [mdn/mdn-community#530] [Content]
https://developer.mozilla.org/en-US/docs/Learn/CSS/Styling_text

I was hoping the examples would provide a pangram. In order to notice any small changes to the letters when making styling changes.

Such as:

The brown fox jumps over a lazy dog.

**Comment by MrBrain295** [comment]
A better one would be "The quick brown fox jumps over the lazy dog."

**Reply by popados** [reply]
opps wrote the wrong sentence! 

Absolutely what I meant! It would make it much easier for all the text styling documentation to have a pangram.

**Comment by pepelsbey** [comment]
Thank you for the suggestion. Feel free to send a PR to the [mdn/content](https://github.com/mdn/content/) if you think you know how to improve text styling examples.

Meanwhile, I’m closing this one as there’s a good chance that the discussion became outdated/irrelevant in the past few years.
